dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -158,7 +159,15 @@ internal class RumViewScope(
     ) {
         delegateEventToChildren(event, writer)
 
-        if (stopped || activeActionScope != null) return
+        if (stopped) return
```

Comment:
no logging (even verbose) in this case to be consistent with other `onXXX` methods in this class

Reply:
Yes it can happen too often that the previous view is still present but stopped.